src/no_mixer.rs

```diff
@@ -1,23 +1,15 @@
-use librespot_playback::mixer::{AudioFilter, Mixer, MixerConfig};
+use librespot_playback::mixer::{Mixer, MixerConfig};
 
 pub struct NoMixer {}
```

Comment:
Only a minor nitpick:

```suggestion
pub struct NoMixer;
```

Didn't think of that in the other PR.